feature: add svg files

OlesiaTr opened the pull request.

OlesiaTr committed:
feature: add svg files

MaksVibe reviewed (changes requested):
There are only 2 svgs for the landing: check, close
Please remove else

OlesiaTr committed:
fix: remove unneeded svg files

OlesiaTr commented:
> There are only 2 svgs for the landing: check, close Please remove else

There are 3 svg files, arrowDown icon should be used for selector and should be rotated when clicked on it. I removed other files, that I added.
